op-node/rollup/clsync/clsync.go

```diff
@@ -117,7 +134,45 @@ func (eq *CLSync) Proceed(ctx context.Context) error {
 		eq.unsafePayloads.Pop()
 		return err
 	}
+
+	if eq.publishAttributes {
+		err = eq.PublishAttributes(ctx, ref)
+		if err != nil {
+			eq.log.Warn("Error publishing L2 attributes", "err", err)
+		}
+	}
+
 	eq.unsafePayloads.Pop()
 	eq.log.Trace("Executed unsafe payload", "hash", ref.Hash, "number", ref.Number, "timestamp", ref.Time, "l1Origin", ref.L1Origin)
 	return nil
 }
+
+func (eq *CLSync) PublishAttributes(ctx context.Context, l2head eth.L2BlockRef) error {
+	l1Origin, err := eq.l1OriginSelector.FindL1Origin(ctx, l2head)
+	if err != nil {
+		eq.log.Error("Error finding next L1 Origin", "err", err)
```

Comment:
is there a need to log errors here when its logged in https://github.com/flashbots/optimism/pull/4/files#diff-606b28086f5f2338fa6ed7aabc59df9d304de4c88d2cbabc7228be5da9592abaR141

Reply:
e1c790a6c4bfb1598d4a2ac48b659c2746d049be